index.html

```diff
@@ -27,6 +27,8 @@
   </section>
   <section data-include="sections/introduction.md" data-include-format="markdown">
   </section>
+  <section data-include="sections/summary.md" data-include-format="markdown">
+  </section>
   <section data-include="sections/jwe.md" data-include-format="markdown">
   </section>
   <section data-include="sections/signing.md" data-include-format="markdown">
```

Comment:
Voor een follow-up PR: laten we dit als sectie 4.2 doen. Volgens mij is er geen extra hoofdstuk nodig.